Content.Shared/CCVar/CCVars.cs

```diff
@@ -7,6 +7,22 @@ namespace Content.Shared.CCVar
     [CVarDefs]
     public sealed class CCVars : CVars
     {
+        /*
+         * Ambience
+         */
+
+        /// <summary>
+        /// How long we'll wait until re-sampling nearby objects for ambience.
+        /// </summary>
+        public static readonly CVarDef<float> AmbientCooldown =
+            CVarDef.Create("ambience.cooldown", 1f, CVar.REPLICATED | CVar.SERVER);
```

Comment:
is the pop-in for sounds bad with the cooldown  this high? (alternatively is the perf bad if the cooldown is lower <1sec?)

Reply:
0.5 is probably okay, I haven't profiled it so probably need to check